grakn-engine/src/main/java/ai/grakn/engine/session/GraqlSession.java

```diff
@@ -122,7 +122,6 @@
     private void refreshGraph() {
         if (graph != null && !graph.isClosed()) graph.close();
         graph = factory.open(GraknTxType.WRITE);
-        graph.showImplicitConcepts(showImplicitTypes);
```

Comment:
There's still a variable `showImplicitTypes` here. We need to kill it and all it's friends and family (that means the flag in `GraqlShell`).